ramls/inventory.raml

```diff
@@ -1,6 +1,6 @@
 #%RAML 1.0
 title: Inventory API
-version: v10.10
+version: v10.11
```

Comment:
A corresponding change should be made to the provided interfaces in the module descriptor

Reply:
Ha, thank you Marc. Trips this developer up every other time. Done.